app/src/androidTest/java/org/mozilla/fenix/ui/SearchTest.kt

```diff
@@ -54,13 +50,11 @@ class SearchTest {
 
     @Test
     fun shortcutButtonTest() {
-        homeScreen { }.dismissOnboarding()
-
         homeScreen {
         }.openSearch {
             verifySearchWithText()
             clickDuckDuckGoEngineButton()
-            typeSearch()
+            typeSearch("mozilla")
```

Comment:
Just an FYI for future reference, for search results this is really the only way we can do this (i.e. this is perfect) as we want to make sure results coming back are visible in the UI.  However, for any other test, we always want to use a mock webserver instead.